src/preload.py

```diff
@@ -613,18 +615,29 @@ def write_apps_json(data, output):
         json.dump(data, f, indent=4, sort_keys=True)
 
 
-def replace_splash_image(image=None):
+def replace_splash_image(balenaOS_version="", image=None):
```

Comment:
NIT: `balena_os_version` would look better.

There is no need for a default value, get_baleaOS_version always returns a string.